tools/block-generator/generator/generate.go

```diff
@@ -611,6 +606,13 @@ func getAssetTxOptions() []interface{} {
 	return []interface{}{assetCreate, assetDestroy, assetOptin, assetXfer, assetClose}
 }
 
+func getAppTxOptions() []interface{} {
```

Comment:
Do these need to be combined? Might be easier to consider them different top level things

Reply:
From a UI perspective, I think it's fine to combine them all here. That's because you can control which app you want to emphasize via higher level weights. EG modified [from the follow-up PR](https://github.com/algorand/go-algorand/blob/f5fd2cea84c04c25ba3277b808e96674ea1f59ea/tools/block-generator/scenarios/config.app.create.yml):

```yml
name: "App Create"
tx_per_block: 100

# transaction distribution
tx_asset_fraction: 0.5
tx_app_fraction: 0.5

# asset config
tx_asset_create_fraction: 1.0

# app kind config
app_swap_fraction: 0.5
app_boxes_fraction: 0.5

# app swap config
app_swap_create_fraction: 1.0

# app boxes config
app_boxes_create_fraction: 0.02
app_boxes_optin_fraction: 0.98
```